sources/Z.100-201811-AnnF1.adoc

```diff
@@ -341,9 +340,9 @@ The following notational conventions are used when declaring names.
 
 * _Domain names_ start with an uppercase letter and presented in small-capitalized italics (as in [smallcap]#_Agent_#), except when denoting a non‑terminal of the SDL‑2010 abstract grammar. In that case, domain names are written as the SDL‑2010 non‑terminals, i.e., in italics, hyphenated, and starting with an uppercase letter (as in _Agent-definition_). A domain name _D_ is declared by *domain* _D_. A domain name is interpreted as a unary predicate which yields _true_ for the members of the base set that belong to the domain.
```

Comment:
Here we should differentiate math variables like `_s_` with emphasized names like `_Domain names_`.

Reply:
Fixed.